examples/combined_ODNP.py

```diff
@@ -62,8 +62,12 @@
 config_dict['FIR_rep'] = FIR_rep
 # }}}
 # {{{Power settings
-dB_settings = gen_powerlist(
-    config_dict["max_power"], config_dict["power_steps"] + 1, three_down=True
+dB_settings = Ep_spacing_from_phalf(
+    est_phalf = config_dict['guessed_phalf'],
+    max_power = config_dict["max_power"], 
+    config_dict["power_steps"] + 1, 
+    min_dBm_step = config_dict['min_dBm_step'],
+    three_down=True
```

Comment:
I think that this is good, but I ended up looking at the function for Ep_spacing_from_phalf --> can you give a link to where that math is on slack?  i.e. how do we justify the math there?

Reply:
I'm saying that I want that info so that I can add the math to the docstring (or else this will never get done)

Reply:
https://jmfrancklab.slack.com/archives/CLMMYDD98/p1668103766923999

Reply:
https://jmfrancklab.slack.com/archives/CLMMYDD98/p1680712474001329?thread_ts=1668103766.923999&cid=CLMMYDD98